var/spack/repos/builtin/packages/elfutils/package.py

```diff
@@ -34,14 +34,14 @@ class Elfutils(AutotoolsPackage):
     version of elfutils."""
 
     homepage = "https://fedorahosted.org/elfutils/"
+    url = 'https://sourceware.org/elfutils/ftp/0.163/elfutils-0.163.tar.bz2'
+
+    version('0.163', '77ce87f259987d2e54e4d87b86cbee41')
 
     depends_on('libtool', type='build')
     depends_on('automake', type='build')
     depends_on('autoconf', type='build')
```

Comment:
Nope. They were probably needed when we were checking out a tag straight from git, but the tarball we are switching to comes with a `configure` script, so these dependencies are no longer needed to generate one.